_templates/playground/new/index.js

```diff
@@ -5,60 +5,80 @@ const changeCase = require('change-case');
 //
 module.exports = {
   prompt: ({ inquirer }) => {
-    const questions = [
-      {
-        type: 'input',
-        name: 'name',
-        message: 'Which component is this playground for?',
-        initial: 'ion-button',
-        validate(value) {
-          return value.match(/^ion-[a-z/-]*[a-z]+$/) ? true : "Component name must be kebab-case and begin with 'ion-'";
+    return inquirer
+      .prompt([
+        {
+          type: 'toggle',
+          name: 'is_component',
+          message: 'Is this playground for a component?',
+          initial: true,
         },
-      },
-      {
-        type: 'input',
-        name: 'path',
-        message: 'What should the playground path be?',
-        hint: 'e.g. `basic` or `theming/colors`',
-        validate(value) {
-          return value.match(/^[a-z]+[a-z/-]*[a-z]+$/)
-            ? true
-            : "Path should begin and end with a letter and only contain lowercase letters, '-', or '/'";
-        },
-      },
-      {
-        type: 'select',
-        name: 'version',
-        message: 'Select the Ionic Framework version for the playground',
-        initial: '7',
-        choices: ['6', '7'],
-      },
-      {
-        type: 'toggle',
-        name: 'css',
-        message: 'Generate custom CSS files?',
-        enabled: 'Yes',
-        disabled: 'No',
-      },
-      {
-        type: 'toggle',
-        name: 'angular_ts',
-        message: 'Generate an Angular TypeScript file?',
-        enabled: 'Yes',
-        disabled: 'No',
-      },
-    ];
+      ])
+      .then((answers) => {
+        return inquirer
+          .prompt([
+            // ask a different question for components vs. other playgrounds
+            answers.is_component
+              ? {
+                  type: 'input',
+                  name: 'component',
+                  message: 'Which component is this playground for?',
+                  initial: 'ion-button',
+                  validate(value) {
+                    return value.match(/^ion-[a-z/-]*[a-z]+$/)
+                      ? true
+                      : "Component name must be kebab-case and begin with 'ion-'";
+                  },
+                }
+              : {
+                  type: 'input',
+                  name: 'name',
+                  message: 'What is this playground for?',
```

Comment:
```suggestion
                  message: 'Which guide section is this playground for?',
```

I think it's probably safe to say this has to go in a guide if it isn't a component, right?